Beyond08-SOW.md

```diff
@@ -228,26 +227,26 @@ You can pick from the ideas below or think up your own.
 
 | #      | Date                    | Topic(s)                                        | Lecturer        |
 |--------|-------------------------|-------------------------------------------------|-----------------|
```

Comment:
Maybe we want to add a location?
like: TelHai Collage, Red Hat office, Virtual Meeting, Margalit office

```suggestion
| #      | Date                    | Topic(s)                                        | Lecturer        | Location
|--------|-------------------------|-------------------------------------------------|-----------------|-----------------| 
```

Reply:
> Maybe we want to add a location? like: TelHai Collage, Red Hat office, Virtual Meeting, Margalit office

Added at the last Column